labshare-client/src/backend/mail/localization.ts

```diff
@@ -6,33 +6,33 @@ export default {
       subject: "LabHive - Account activation",
       text: (link: string) => {
         return `
-Hallo, 
-um Ihren Account auf LabHive zu aktivieren, verwenden Sie bitte folgenden Link:
+Dear user, 
+to activate your account on LabHive, please use the following link:
```

Comment:
In English, the subsequent line after the salutation in a letter is capitalised. (Yes, even though it's opened with a comma. Yes, I know that's weird, but it was also weird learning it in the other direction.)